system-tests/__snapshots__/stdout_spec.js

```diff
@@ -275,15 +275,15 @@ exports['e2e stdout displays fullname of nested specfile 1'] = `
   ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
   │ Cypress:    1.2.3                                                                              │
   │ Browser:    FooBrowser 88                                                                      │
-  │ Specs:      3 found (spec.cy.js, stdout_specfile.cy.js, stdout_specfile_display_spec_with_a_re │
-  │             ally_long_name_that_never_has_a_line_break_or_new_line.cy.js)                      │
-  │ Searched:   cypress/e2e/nested-1/nested-2/nested-3/*                                           │
+  │ Specs:      4 found (spec.cy.js, stdout_specfile.cy.js, stdout_specfile_display_spec_with_a_re │
```

Comment:
I was expecting a lot of snapshots to break but no - the directory structure for all our system-tests projects is very flat so we didn't notice when this regressed - only this one had any changes.

I guess this one snapshot got lost/missed in the original review that caused this regression in the first place.